### PR TITLE
Tbl directly use uaa sign in

### PR DESCRIPTION
**Pull request by tej99**
TBL to get response operations to directly sign in using UAA instead of through backstage.

## How to Test
- Check if you can successfully sign into internal account (uaa_user)
- Check you get error banner if wrong password entered
- Check you can view your secure messages once logged in

## Config updates

Updated confluence: https://digitaleq.atlassian.net/wiki/spaces/RASB/pages/406257665/Pending+Release+Config+Changes
Updated docker dev: https://github.com/ONSdigital/ras-rm-docker-dev/pull/54

Updated concourse pipeline: https://github.com/ONSdigital/ras-deploy/pull/9

https://trello.com/c/oodNbXR2/100-response-operations-ui-should-sign-in-directly-using-uaa-not-via-backstage

### CODE REVIEW

**Comment by tej99** [comment]
[response-operations-ui should sign in directly using uaa (not via backstage)](https://trello.com/c/oodNbXR2/100-response-operations-ui-should-sign-in-directly-using-uaa-not-via-backstage)

File: tests/views/test_sign_in.py [hunk 1]
```diff
@@ -38,9 +47,19 @@ def test_sign_in(self, mock_request):
...
+        response = self.app.post("/sign-in", follow_redirects=True,
+                                 data={"username": "user", "password": "pass"})
+
+        self.assertEqual(response.status_code, 500)
+        self.assertIn(b'Error 500 - Server error', response.data)
+
     @requests_mock.mock()
     def test_fail_authentication(self, mock_request):
-        mock_request.post(url_sign_in_data, json={"token": "1234abc"}, status_code=401)
+        mock_request.post(url_sign_in_data, json={}, status_code=200)  # No token in response
```
Comment (AdamHawtin) [inline tests/views/test_sign_in.py]: This seems to be a test for a failure case, I don't think mocking UAA returning a 200 is appropriate here.

**Comment by jcox-dev** [comment]
[response-operations-ui should directly call uaa service (not via backstage)](https://trello.com/c/LfVEOyC8/128-response-operations-ui-should-directly-call-uaa-service-not-via-backstage)

File: tests/views/test_sign_in.py (hunk 1 above)
Comment (tej99) [reply tests/views/test_sign_in.py]: UAA could give back a successful response without a token (a test case being tested in backstage). So the success status code makes sense in this case.

Comment (tej99) [reply tests/views/test_sign_in.py]: Sorry misunderstood, I have added the missing 401 UAA response test case and renamed the test for the missing token 👍

**Pull request merged**